google/cloud/forseti/notifier/notifier.py

```diff
@@ -84,7 +84,7 @@ def process(message):
     payload = message.get('payload')
 
     if message.get('status') == 'inventory_done':
-        inv_email_pipeline = inv_summary.EmailInventorySnapshotSummaryPipeline(
+        inv_email_pipeline = inv_summary.EmailInventorySnapshotSummary(
```

Comment:
You may as well remove `pipeline` from `inv_email_pipeline`.

Reply:
done